### DISCUSSION TITLE
Interface

### DISCUSSION THREAD

**Discussion by nisancigokmen** [fonoster/yaps#222] [Ideas]
The interface part can be improved a bit. I would like to help

**Comment by psanders** [comment]
@nisancigokmen 

We are talking CLI, correct?

**Reply by nisancigokmen** [reply]
@psanders  I'm talking about the front end part...

**Reply by psanders** [reply]
It needs a lot of work :)

**Reply by nisancigokmen** [reply]
@psanders  I will support. good work   :)

**Reply by AlijiEmmanuel-dev** [reply]
I will like to support. Great work.
